packages/loaders/code-file/src/index.ts

```diff
@@ -59,7 +58,7 @@ const FILE_EXTENSIONS = [
   '.gjs',
 ];
 
-function createGlobbyOptions(options: CodeFileLoaderOptions): GlobbyOptions {
+function createGlobbyOptions(options: CodeFileLoaderOptions): GlobOptions {
```

Comment:
Are we sure the options are the same? Otherwise it is a breaking change.

Reply:
It looks like the options used here are `cwd, `absolute`, and `ignore`. The `tinyglobby` package supports all three of those options